Libraries/LibGUI/Clipboard.cpp

```diff
@@ -62,6 +64,25 @@ Clipboard::DataAndType Clipboard::data_and_type() const
     return { data, type };
 }
 
+AK::RefPtr<Gfx::Bitmap> Clipboard::get_bitmap(Gfx::BitmapFormat format, Gfx::Size size)
```

Comment:
No need to qualify `AK::` basically ever, just say `RefPtr`.

More importantly, it seems unhealthy that the caller has to provide both `format` and `size`.
The caller should only have to say "gimme a bitmap" and the `Clipboard` class either says "OK here u go" or `nullptr`.